css/style.css

```diff
@@ -748,20 +748,23 @@ body {
   font-size: 20px;
 }
 
-.cards .contact {
+.contact {
+  background: #6070ff;
+  background-image: url(../contact\ form\ shapes-mobile@2x.png);
   width: 375px;
```

Comment:
Great work fixing a mobile width for your website. ↔️  
While it is a cool idea, it is not a recommended way to set width as now all modern websites are meant to be responsive and have a similar display on all screens. 📱 💻 🖥️ 

- [ ] I would suggest that you remove all the fixed widths given to the various sections and make them responsive as suggested by the previous reviewer. You can make use of `responsive units` like `%` or `vh and vh` etc ♻️ 

### Currently, your website does not respond to any screen size above 375px ⬇️ 
![responsive ](https://github.com/jalal1379/Microverse_portflaio/assets/117144028/2d3490ea-62bf-46b3-b4c7-30337ebd390f)